crates/ra_assists/src/assists/flip_trait_bound.rs

```diff
@@ -0,0 +1,184 @@
+//! `flip_trait_bound` provides an assist that flips the positions of adjacent
+//! type bounds in a type bound list.
+
+use crate::{Assist, AssistCtx, AssistId};
+use hir::db::HirDatabase;
+use ra_syntax::ast::{AstNode, TypeBound, TypeBoundList};
+use ra_syntax::{NodeOrToken, TextRange};
+use std::cmp::{max, min};
+
+/// Flip trait bound assist.
```

Comment:
That's a new addition, but, instead of writing a doc comment here, one should use a special form of a non-doc comment, which is used by `cargo xtask codegen` to automatically generate tests & docs for an assist. 

See this for an example: https://github.com/rust-analyzer/rust-analyzer/blob/733fd64260793a0f7335e4f75ba9197d5fa98b70/crates/ra_assists/src/assists/flip_binexpr.rs#L6-L20